server/data-service/src/main/java/edu/illinois/ncsa/incore/service/data/models/Dataset.java

```diff
@@ -406,18 +406,8 @@ public String toString() {
      *
      * @return id of the bean
      */
-    public final String getId() {
-        return id.toString();
-    }
-
-    /**
-     * Sets the id of the bean. This has to be a unique id since it is used as
-     * the key in the database.
-     *
-     * @param id the id of the object.
-     */
-    public void setId(String id) {
```

Comment:
Why remove this? Just curious, but the change could have been more concise even if this "should" get removed eventually if we intend to not allow it.